docs/api/ShallowWrapper/prop.md

```diff
@@ -2,24 +2,21 @@
 
 Returns the prop value for the node of the current wrapper with the provided key.
 
-NOTE: can only be called on a wrapper of a single node.
+NOTE: can only be called on a wrapper of a single node. Cannot be called on the root wrapper because that represents what the component renders, not the component itself.
 
 #### Arguments
 
 1. `key` (`String`): The prop name such that this will return value will be the `this.props[key]`
 of the component instance.
 
-
-
 #### Example
 
 
 ```jsx
-const wrapper = shallow(<MyComponent foo={10} />);
-expect(wrapper.prop('foo')).to.equal(10);
+const wrapper = shallow(<div><MyComponent foo={10} /></div>);
+expect(wrapper.find('MyComponent').prop('foo')).to.equal(10);
```

Comment:
:+1: on demonstrating this with a prop on the root node of `MyComponent`.

About the second one, I get "TypeError: wrapper.instance(...).prop is not a function" which is to be expected as `.instance()` returns the ReactComponent, not the wrapper and `.prop()` is a function on the wrapper. Could you clarify what you meant by it?